build.xml

```diff
@@ -186,6 +186,7 @@
       doctitle="${name} ${version} API"
       bottom="Copyright &amp;copy; ${year} The Apache Software Foundation"
       failonerror="true"
+      failonwarning="true"
```

Comment:
Observed two warnings when running `ant clean javadoc` (which made the build to fail):
``` 
  [javadoc] .../src/java/org/apache/nutch/protocol/RobotRulesParser.java:58: warning - Tag @link: reference not found: getRobotRulesSet
  [javadoc] .../src/plugin/indexer-elastic/src/java/org/apache/nutch/indexwriter/elastic/ElasticIndexWriter.java:204: warning - Tag @link: reference not found: BulkProcessor.Listener
  ...
  [javadoc] 2 warnings

BUILD FAILED
.../build.xml:723: Javadoc issued warnings.
```

Reply:
Hi @sebastian-nagel in short no I am not able to reproduce.
```% ant clean runtime
...
  [javadoc] Standard Doclet version 1.8.0_221
  [javadoc] Building tree for all the packages and classes...
  [javadoc] Building index for all the packages and classes...
  [javadoc] Building index for all classes...
     [copy] Copying 1 file to /Users/lmcgibbn/Downloads/nutch/build/docs/api/org/apache/nutch/plugin/doc-files
     [copy] Copying 1 file to /Users/lmcgibbn/Downloads/nutch/build/docs/api/resources
     [copy] Copying 1 file to /Users/lmcgibbn/Downloads/nutch/build/docs/api/resources

BUILD SUCCESSFUL
Total time: 38 seconds
```

Reply:
It looks like CI was not able to reproduce either...